projects/packages/jetpack-mu-wpcom/src/features/domain-email-nag/domain-email-nag.php

```diff
@@ -63,6 +63,12 @@ function domain_email_nag() {
 		return;
 	}
 
+	$pixel_url = sprintf(
+		'http://pixel.wp.com/g.gif?v=wpcom-no-pv&x_%s=%s',
```

Comment:
I prefer using `b.gif` which does the same thing but renders a transparent pixel rather than a smiley face. I'm always paranoid that there'll be some error that causes the image to appear really big or something, and the smiley face would feel inappropriate then 😅 that's me being paranoid though.

I'm not seeing any of these stats appear in MC yet. I would have thought I'd see something from testing (even if I personally have only just managed to get the nag showing).

Maybe that's because we need to change the URL to something like. `&x_wpcom_frontend_unverified_domain_email_nag=shown`, with an equals in there. I don't see why the group and event name can't be the same. But maybe it can't and I just remembered that this way of doing things (separating out the verb into a separate name) is more common. So maybe it's required or something.

Reply:
TIL. Thanks for this change.